src/setup.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+// const { User } = require("./models/User.js");
+// const { Token } = require('./models/Token.js');
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)